scripts/generator/graphql_generator/csharp/SDK/Android/AndroidNativeCheckout.cs.erb

```diff
@@ -16,10 +16,28 @@ namespace <%= namespace %>.SDK.Android {
         private String MerchantName;
         private String CountryCodeString;
         private AndroidPayEventReceiverBridge AndroidPayEventBridge;
+        #if !SHOPIFY_MONO_UNIT_TEST
+        private AndroidJavaObject _androidPayCheckoutSession;
+        private AndroidJavaObject AndroidPayCheckoutSession {
+            get {
+                if (_androidPayCheckoutSession == null) {
+                    const string target = "com.shopify.unity.buy.androidpay.AndroidPayCheckoutSession";
```

Comment:
Nit: I would probably move this up to the class as a `const` string just to make it more obvious when looking at this source which class we're creating.

Reply:
Fair enough!

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/360/commits/b922c0be01824c8ce9ff43eb19506937080002fb